lib/misc/api/api_logon.dart

```diff
@@ -0,0 +1,126 @@
+import 'dart:async';
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+import 'api_url.dart' as api;
+import 'package:flutter/foundation.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+
+// Create the storage
+const storage = FlutterSecureStorage();
+
+/// Store API token
+storeToken(String value) async {
+  await storage.write(key: "token", value: value);
+}
+
+/// Store login information
+storeLoginInfo(String email, String password) async {
+  await storage.write(key: "email", value: email);
+  await storage.write(key: "password", value: password);
+}
```

Comment:
Should for safety reason probably only store token and update its expiration date

Reply:
Same as previous